### PR TITLE
Font size to match design

### PR DESCRIPTION
**Pull request by ben2x4**

### CODE REVIEW

File: components/SwapForm/SwapFormStyles.tsx [hunk 1]
```diff
@@ -3,10 +3,11 @@ import styled from 'styled-components'
 import { SegmentedController } from '../SegmentedController'
 
 export const SwapFormHeading = styled(Text).attrs(() => ({
-  type: 'heading',
+  type: 'title',
   variant: 'normal',
 }))`
   padding: 44px 0 32px;
+  margin: 15px;
```
Comment (vagabundsf) [inline components/SwapForm/SwapFormStyles.tsx:10]: This is adding horizontal offset; is that intended? 
```suggestion
  margin: 15px 0;
```

**Review by vagabundsf — COMMENTED** [review]
Looks great! Leaving a minor question about the margin

**Review by vagabundsf — APPROVED** [review]
Ohh I see it's matching up the specs now!

Lgtm

File: components/SwapForm/SwapFormStyles.tsx (hunk 1 above)
Comment (ben2x4) [reply components/SwapForm/SwapFormStyles.tsx:10]: yup, its like this in the designs and I think it looks better

**Pull request merged**